topostats/plotting_dictionary.yaml

```diff
@@ -109,27 +109,32 @@ removed_small_objects:
   title: "Small Objects Removed"
   image_type: "binary"
   core_set: false
+grains_removed_based_on_height:
```

Comment:
I think that `remove_small_objects` may be best deleted, since so much of what it does overlaps with `area_thresholding`, so I think I'll leave this to discussion and a different PR if that's okay